### PR TITLE
797318 - Amounts ending in zero displayed as fractions

### PR DESCRIPTION
**Pull request by jeanlaroche**
The function gnc_numeric_to_decimal() should return TRUE if num is 0 but currently does not.

### CODE REVIEW

File: libgnucash/engine/iso-4217-currencies.xml [hunk 1]
```diff
@@ -1744,7 +1744,7 @@
   namespace="ISO4217"
   exchange-code="478"
   parts-per-unit="5"
-  smallest-fraction="100"
+  smallest-fraction="5"
```
Comment (jralls) [inline libgnucash/engine/iso-4217-currencies.xml:1747]: We follow [ISO-4217](https://www.currency-iso.org/dam/downloads/lists/list_one.xml) and it says
```
<CcyNtry>
<CtryNm>MAURITANIA</CtryNm>
<CcyNm>Ouguiya</CcyNm>
<Ccy>MRU</Ccy>
<CcyNbr>929</CcyNbr>
<CcyMnrUnts>2</CcyMnrUnts>
</CcyNtry>
```
So smallest fraction stays at 100. Users may set 5 in accounts if they like.

Comment (jeanlaroche) [reply libgnucash/engine/iso-4217-currencies.xml:1747]: OK I"ll revert that change, but the rest can stay in case the user changes it, right?

Comment (jralls) [reply libgnucash/engine/iso-4217-currencies.xml]: Yes, the bit about returning true on 0 is a good catch indeed.

**Comment by jralls** [comment]
Thanks!

**Pull request merged**